examples/guestbook/README.md

```diff
@@ -513,65 +580,27 @@ if (isset($_GET['cmd']) === true) {
 
 Note the use of the `redis-master` and `redis-slave` host names-- we're finding those services via the Kubernetes cluster's DNS service, as discussed above.  All the frontend replicas will write to the load-balancing redis-slaves service, which can be highly replicated as well.
 
+### Step Seven: Cleanup
 
-### Step Six: Set up the guestbook frontend service.
-
-As with the other pods, we now want to create a service to group your frontend pods.
-The service is described in the file `frontend-service.yaml`:
-
-<!-- BEGIN MUNGE: EXAMPLE frontend-service.yaml -->
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: frontend
-  labels:
-    name: frontend
-spec:
-  # if your cluster supports it, uncomment the following to automatically create
-  # an external load-balanced IP for the frontend service.
-  # type: LoadBalancer
-  ports:
-    # the port that this service should serve on
-    - port: 80
-  selector:
-    name: frontend
-```
-
-[Download example](frontend-service.yaml?raw=true)
-<!-- END MUNGE: EXAMPLE frontend-service.yaml -->
-
-#### Using 'type: LoadBalancer' for the frontend service (cloud-provider-specific)
-
-For supported cloud providers, such as Google Compute Engine or Google Container Engine, you can specify to use an external load balancer
-in the service `spec`, to expose the service onto an external load balancer IP.
-To do this, uncomment the `type: LoadBalancer` line in the `frontend-service.yaml` file before you start the service.
-
-[See the section below](#accessing-the-guestbook-site-externally) on accessing the guestbook site externally for more details.
-
-
-#### Create the Frontend Service ####
-
-Create the service like this:
+If you are in a live kubernetes cluster, you can just kill the pods by stopping the replication controllers and deleting the services.  Using labels to select the resources to stop or delete is an easy way to do this in one command.
```

Comment:
Should using labels to delete resources, if we add the all-in-one.yaml in the same directory.